.changeset/smart-lobsters-jump.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Note, documentation will follow in another PR. Wanted to keep this PR focused on code changes.